docs/Additional Information/workshop.md

```diff
@@ -32,7 +32,7 @@ Ropsten Explorer: <a href="https://ropsten.explorer.chain.link/" target="_blank"
 - Ropsten LINK faucet: <a href="https://ropsten.chain.link" target="_blank">https://ropsten.chain.link</a>
 
 ## Slide: Add the LINK token to Metamask
-[block:code]
+```
```

Comment:
You can't just replace [block:code] with three backticks, you need to actually take the code out of the JSON to make sure it shows up correctly. It's not supposed to look like JSON, it's just supposed to be the code.

Reply:
noted